src/Pages/LoginPage.tsx

```diff
@@ -74,65 +69,60 @@ export const LoginPage: React.FC<LoginPageProps> = ({
-                    serviceTerms='terms & conditions'
-                    href={termsAndConditionLink}
-                    isChecked={isChecked}
-                    onChange={onChangeCheckbox}
-                  />
-                </LoginWrapper>
-                <RegistrationButtonContainer>
-                  <Button
-                    isPrimary={true}
-                    label='Login Account'
-                    onClick={onClickLogin}
-                    type='submit'
-                  />
+          <RegistrationOnboardingContent>
+            <HeaderStyle>
+              <Back href={backLink} label='Back' />
+              <HeaderContent>
+                <Login isSignedUp={false} href='./registered' />
+              </HeaderContent>
+            </HeaderStyle>
+            <RegistrationForm onSubmit={onSubmit}>
+              <RegistrationFormHeading>
+                Log in your account
+              </RegistrationFormHeading>
+              <RegistrationFormContent>
+                <Input
+                  label='Email address'
+                  placeholder='Enter email address'
+                  inputType='email'
+                  inputId='emailAddress'
+                  inputValue={accountLogin?.emailAddress}
+                  onChange={onChangeEmail}
+                  errorMessage={errorMessage?.email}
+                />
+                <Input
+                  label='Create password'
+                  placeholder='Enter a password'
+                  inputType='password'
+                  inputId='password'
+                  inputValue={accountLogin?.password}
+                  onChange={onChangePassword}
+                  errorMessage={errorMessage?.password}
+                />
+              </RegistrationFormContent>
+              <TermsAndConditions
+                termsLabel='I agree to the'
+                serviceTerms='terms & conditions'
+                href={termsAndConditionLink}
+                isChecked={isChecked}
+                onChange={onChangeCheckbox}
+              />
+              <RegistrationButtonContainer>
+                <Button
+                  isPrimary={true}
+                  label='Login Account'
+                  onClick={onClickLogin}
+                  type='submit'
+                />
 
-                  <Button
-                    type='button'
-                    label='Login with Google'
-                    buttonIcon={ButtonIcon}
-                  />
-                </RegistrationButtonContainer>
-              </RegistrationForm>
-            </RegistrationOnboardingContent>
-          </RegistrationOnboardingContainer>
+                <Button
+                  type='button'
+                  label='Login with Google'
+                  buttonIcon={ButtonIcon}
+                />
+              </RegistrationButtonContainer>
+            </RegistrationForm>
+          </RegistrationOnboardingContent>
```

Comment:
Why registration is in Login page ?

Reply:
It should be renamed to login. I see its not part of your PR so live it as it is just add todo at the top that it needs to be renamed please. Approving in advance